components/brave_wallet/browser/polkadot_keyring_unittest.cc

```diff
@@ -0,0 +1,161 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/polkadot_keyring.h"
+
+#include <string_view>
+
+#include "base/strings/string_number_conversions.h"
+#include "brave/components/brave_wallet/common/encoding_utils.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace brave_wallet {
+namespace {
+
+// All derived from the polkadot-sdk using:
+// let pair =
+//   polkadot_sdk::sp_core::sr25519::Pair::from_string(
+//     DEV_PHRASE, None).unwrap();
+
+inline constexpr const char* kDevPhrase =
+    "bottom drive obey lake curtain smoke basket hold race lonely fit walk";
+
```

Comment:
`~inline constexpr const char* kDevPhrase~`
This is typically in form of:
`constexpr const char kDevPhrase[]`

Reply:
Ha, didn't even realize. This is done. Thank you!